tests/Core/Components/Switch/FluentSwitchTests.razor

```diff
@@ -0,0 +1,80 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using System.ComponentModel.DataAnnotations
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentSwitchTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentSwitch_Default()
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentSwitch Autofocus="true"
+                AriaLabel="My aria label"
+                Name="MyName"
+                Value="true" />
+        );
+        // Assert
+        cut.Verify();
+    }
+
+    [Fact]
+    public void FluentCheckbox_LabelPosition()
+    {
+        // Arrange && Act
+        var cut = Render(
+            @<FluentSwitch LabelPosition="LabelPosition.Above" />
+        );
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Fact]
+    public void FluentCheckbox_LabelTemplate()
+    {
+        // Arrange && Act
+        var cut = Render(
+            @<FluentSwitch Label="Title" />
+        );
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Fact]
+    public void FluentCheckbox_Required()
+    {
+        // Arrange && Act
+        var cut = Render(
+            @<FluentSwitch Required="true" />
+        );
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Theory]
+    [InlineData(false, true)]
+    [InlineData(true, false)]
+    public void FluentSwitch_OnClick(bool initialValue, bool expectedValue)
+    {
+        // Arrange
+        var value = initialValue;
+        var cut = Render(@<FluentSwitch @bind-value="@value" />);
```

Comment:
You need to use the PascalCase for the parameter: `@bind-Value`

Reply:
ok, changed to `@bind-Value`